assets/app/view/game/stock_market.rb

```diff
@@ -72,6 +72,13 @@ def box_style_1d
         }
       end
 
+      def box_height_1d
+        token_height = @game.stock_market.market.first.map do |p|
+          p.corporations.sum { |c| TOKEN_SIZES[@game.corporation_size(c)] + VERTICAL_TOKEN_PAD }
+        end.push(MIN_NUM_TOKENS * (TOKEN_SIZE + VERTICAL_TOKEN_PAD)).max
```

Comment:
i don't like this style and don't use push

maybe something like
 (map { p.sum } + [constan]).max

Reply:
just copy & pasted, will check later